gem/templates/comments/comment.html

```diff
@@ -36,17 +31,14 @@
 
     <span class="date">{{node.submit_date|timesince}} {% trans "ago" %}</span>
     {% if not node.user|is_in_admin_group %}
-      <p>
-        <a href="{% url 'report_comment' node.pk %}" class="report">{% trans "Report" %}</a>
-      </p>
+      <a href="{% url 'report_comment' node.pk %}" class="report">{% trans "Report" %}</a>
     {% endif %}
 
     {% if node.parent == None %}
         {% if request.user.is_authenticated %}
```

Comment:
@Mitso I think include `and request.user|is_in_admin_group` here, which can be removed once comment threading is implemented. This will ensure that only admin users can reply to comments on the front end.